dm/dm-table-routing.md

```diff
@@ -116,7 +116,7 @@ To extract the source information of upstream sharded tables to the merged table
 
 ```sql
 CREATE TABLE `test`.`t` (
-    a int(11) PRIMARY KEY,
+    a int PRIMARY KEY,
```

Comment:
Similar here I assume MySQL is the upstream, but I have no issues with updating these as well (unless the output is still including the display width from MySQL or DM).